### PR TITLE
Add page documenting vanilla command permissions

### LINKED ISSUE #340: Document vanilla command permissions
Add a documentation about all vanilla commands and their respective permissions. A note whether they are given by default or need extra privileges (eg. op) would also be nice. 
Reference: [outdated wiki page](https://bukkit.fandom.com/wiki/CraftBukkit_Commands)

### PR DESCRIPTION
**Pull request by pontaoski**
Fixes #340.

### CODE REVIEW

File: docs/paper/admin/reference/vanilla-command-permissions.mdx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+slug: /reference/vanilla-command-permissions
+description: Vanilla Minecraft commands and their associated permissions
```
Comment (zlataovce) [inline docs/paper/admin/reference/vanilla-command-permissions.mdx:3]:
```suggestion
description: Vanilla Minecraft commands and their associated permissions.
```

File: docs/paper/admin/reference/vanilla-command-permissions.mdx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+---
+
+# Vanilla Command Permissions
+
+The following is a list of permissions for vanilla commands.
```
Comment (zlataovce) [inline docs/paper/admin/reference/vanilla-command-permissions.mdx:8]:
```suggestion
The following is a list of permissions for Vanilla commands.
```

File: docs/paper/admin/reference/vanilla-command-permissions.mdx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+import React from 'react';
+import { commands } from '@site/src/components/MinecraftCommands'
```
Comment (zlataovce) [inline docs/paper/admin/reference/vanilla-command-permissions.mdx:11]:
```suggestion
import React from "react";
import { commands } from "@site/src/components/MinecraftCommands";
```

File: src/components/MinecraftCommands.ts [hunk 4]
```diff
@@ -0,0 +1,82 @@
+type CommandData = [string, string[], number | string];
```
Comment (zlataovce) [inline src/components/MinecraftCommands.ts:1]: this should rather be an interface with properly named properties, looks a little confusing without description

File: src/components/MinecraftCommands.ts
Comment (zlataovce) [inline src/components/MinecraftCommands.ts:1]: should probably be moved into a different directory, there's no component in the file

**Review by olijeffers0n — APPROVED** [review]

**Review by olijeffers0n — CHANGES_REQUESTED** [review]
Check Vercel Comments Please

**Comment by pontaoski** [comment]
the vercel comments aren't public

**Comment by olijeffers0n** [comment]
> the vercel comments aren't public

Some table cells need restyling to avoid wrap

**Review by olijeffers0n — APPROVED** [review]

**Pull request merged**